tsjs.js

```diff
@@ -33,12 +33,33 @@ try {
         fs.writeFileSync(tempTsConfigFile, JSON.stringify(tsConfigLint));
         tsConfigFile = tempTsConfigFile;
     }
+
+    const program = Linter.createProgram(tsConfigFile, ".");
+    const linter = new Linter({fix: true}, program);
+
+    const files = Linter.getFileNames(program);
+    files.forEach((file) => {
+        const fileContents = program.getSourceFile(file).getFullText();
+        const configuration = Configuration.findConfiguration(tempTsLintFile, file).results;
+        linter.lint(file, fileContents, configuration);
+    });
+

```

Comment:
console.log(`Starting tsfmt with the following configuration:  
tslint: tsjs configuration  
tsfmt: tsjs configuration  
tsconfig: ${tsConfigFile}  
excluded files:
${[...tsConfigLint.exclude, ...excludeOptionArray].join('\n')}`);